File: 1036.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

int main(void) {
    double A, B, C, delta;

    scanf("%lf", &A);
    scanf("%lf", &B);
    scanf("%lf", &C);

    delta = (B*B)-(4*A*C);

    if (A == 0) {
        printf("Impossivel calcular");
    } else if (delta > 0) {
        printf("R1 = %.5lf\n", ((-B + sqrt(delta)) / (2 * A)));
        printf("R2 = %.5lf\n", ((-B - sqrt(delta)) / (2 * A)));
    } else {
        printf("Impossivel calcular");
    }

    return 0;
}

File: 1036.py
import math;
a = float(input("A: "));
b = float(input("B: "));
c = float(input("C: "));

delta = b**2 - 4 * a * c;
if(delta<0 or a == 0):
  print("Impossivel calcular")
else:
  R1 = (-b + (math.sqrt(delta))) / (2 * a);
  R2 = (-b - (math.sqrt(delta))) / (2 * a);
  print('R1 = {:.5f}'.format(R1));
  print('R2 = {:.5f}'.format(R2));